### ISSUE TITLE
Bug iconAssetName doesn't work

### ISSUE THREAD

**Issue by hm21** [khanmujeeb687/flutter_pinned_shortcut#1]
There is a bug when using a custom icon with the option “iconAssetName”. I found out that the problem is in the file “flutter_pinned_shortcut_method_channel.dart”, you only need to replace the following:
```
 return await methodChannel.invokeMethod("createPinnedShortcut", {
      "id": id,
      "shortLabel": label,
      "action": action,
      // Replace this with the next line => "iconAssetName": iconAssetName,
      "icon": iconAssetName, 
    });
```
Alternatively, you can change it in the file “PinnedShortcutUtil.java”.

**Comment by khanmujeeb687** [comment]
Hello Alex,

Thank you so much for reporting this bug. This issue will be fixed in the
next version that will release tomorrow.

**Comment by khanmujeeb687** [comment]
Hey @hm21 The issue has been fixed in version 0.0.4. Happy coding :)
